src/device-api/v2.ts

```diff
@@ -459,6 +458,22 @@ export function createV2Api(router: Router, applications: ApplicationManager) {
 		}
 	});
 
+	router.get('/v2/device/vpn', async (_req, res) => {
+		// TODO: move all response interfaces to single location
+		interface VPNInfo {
```

Comment:
There's no real need to define this interface if it's only used here; typescript will infer it well enough (unless you want to add to it later in the function).

Reply:
I added it to indicate I was planning for it to be extended in the future but can remove for now.

Reply:
Ideally we'd skip this kind of premature scaffolding, a lot of the time these can very easily be added when the time comes.

Reply:
ok sounds good